src/components/Select/PSelect.vue

```diff
@@ -134,6 +130,10 @@
 
   const modelValue = computed({
     get() {
+      if (multiple.value) {
+        return props.modelValue ?? []
+      }
+
```

Comment:
small bugfix. Noticed this component wasn't working correctly with `modelValue: null | undefined` and `multiple: true`